nixos/modules/flyingcircus/roles/mailserver.nix

```diff
@@ -10,6 +10,7 @@ let
     null
     config.flyingcircus.enc_services;
 
+  myHostname = (fclib.configFromFile /etc/local/postfix/myhostname "");
```

Comment:
Strictly speaking, the parentheses are not necessary since there is only one function call in this expression.